tests/fixtures/docker/integration_internal_auth.env

```diff
@@ -22,9 +22,9 @@ API_LOGGING_LEVEL=DEBUG
 FIFTYONE_ENV=development
 CAS_DEBUG="cas:*"
 
-# Image tags
+# Image tags for existing artifacts
 VERSION=v1.6.0
-FIFTYONE_APP_RC=${VERSION}rc14
-FIFTYONE_TEAMS_API_RC=${VERSION}rc13
-FIFTYONE_TEAMS_APP_RC=${VERSION}-rc.11
-FIFTYONE_TEAMS_CAS_RC=${VERSION}-rc.11
+FIFTYONE_APP_VERSION="${VERSION}rc14"
+FIFTYONE_TEAMS_API_VERSION="${VERSION}rc13"
+FIFTYONE_TEAMS_APP_VERSION="${VERSION}-rc.11"
+FIFTYONE_TEAMS_CAS_VERSION="${VERSION}-rc.11"
```

Comment:
The variable values may be RCs, but not necessarily.  So let's rename the vars. And quote them